cmd/documentation/documentation.go

```diff
@@ -168,151 +172,26 @@ func main() {
 		if verbose {
 			fmt.Println("Fetching repository contributor list...")
 		}
-		contributors, err = GetContributorList(config.GithubRepo, verbose)
+		contributors, err = GetContributorList(context.TODO(), config.GithubRepo, verbose)
 		if err != nil {
 			log.Fatalf("Documentation Generation Tool - GetContributorList error %s",
 				err)
 		}
 
-		// Github API missing contributors
+		// Github API missing/deleted user contributors
```

Comment:
🐒 
Now that pagination is fixed, surely it's *just* deleted, and there's no false positives?
```suggestion
		// Github API deleted user contributors
```

Though I think "missing" covers deleted too, fwiw

Reply:
Nope they mean different things because we've done PR's where a person wasn't familiar with opening PR's, so did it for them. They're not technically a deleted user but their contributions were missing.